x-pack/plugins/observability_solution/slo/common/locators/paths.ts

```diff
@@ -25,8 +25,8 @@ export const paths = {
     `${SLOS_BASE_PATH}${SLOS_PATH}/edit/${encodeURIComponent(sloId)}?_a=${encodedParams}`,
   sloDetails: (sloId: string, instanceId?: string) =>
     !!instanceId
-      ? `${SLOS_BASE_PATH}${SLOS_PATH}/${encodeURIComponent(sloId)}?instanceId=${encodeURIComponent(
+      ? `${SLOS_BASE_PATH}/${encodeURIComponent(sloId)}?instanceId=${encodeURIComponent(
```

Comment:
This was adding an extra `/` in the SLO details path (`http://{KIBANA_URL}/app/slos//{SLO_ID}`) of the `Worst performing` link when trying to navigate from the dashboard app to the slo detail page. 

After this fix, worst performing link works fine in both pages (Dashboard app and SLO list page)!

https://github.com/elastic/kibana/assets/2852703/ee72b14a-fd56-4a87-82fa-a30091ca555d